megamek/src/megamek/client/ui/swing/MiniReportDisplay.java

```diff
@@ -86,6 +85,8 @@ public class MiniReportDisplay extends JPanel implements ActionListener, Hyperli
     private static final GUIPreferences GUIP = GUIPreferences.getInstance();
     private static final ClientPreferences CP = PreferenceManager.getClientPreferences();
 
+    private int filterToggle = 0; // set keyword filter to "off"
```

Comment:
Do not use integers for flow control, if you need a boolean `filterEnabled` then use a boolean.